mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -161,6 +163,15 @@ const NEGOTIATE_SEND_INTERVAL: f64 = 30.;
 /// If a certain P2P message is not received, swap will be aborted after this time expires.
 const NEGOTIATION_TIMEOUT_SEC: u64 = 90;
 
+/// Add refund fee to calculate maximum available balance for a swap (including possible refund)
+pub(crate) const INCLUDE_REFUND_FEE: bool = true;
+
+/// Do not add refund fee to calculate fee needed only to make a successful swap
+pub(crate) const NO_REFUND_FEE: bool = false;
```

Comment:
Q, when are each used and why?

Reply:
Both are part of same entity, meaning to include or not the refund fee into the return value.
Does it look confusing?
Idea was to use a named const instead of just true or false.

Reply:
nah, that's not what I meant.

I just ran out of time and decided to ask when each case is used instead of digging in the code 😂

I'm asking here when do we use the fee calcs with refund in mind and when do we use it without considering a possible refund. and why would we use each in such cases.

Reply:
nah, that's not what I meant.

I just ran out of time and decided to ask when each case is used instead of digging in the code 😂

I'm asking here when do we use the fee calcs with refund in mind and when do we use it without considering a possible refund. and why would we use each in such cases.